src/types/assertionRef.ts

```diff
@@ -0,0 +1,59 @@
+import * as t from "io-ts";
+import { JsonFromString } from "io-ts-types";
+
+import { AssertionRef } from "../../generated/lollipop-api/AssertionRef";
+
+// StoredAssertionRefV1
+export type StoredAssertionRefV1T = t.TypeOf<typeof StoredAssertionRefV1>;
+export const StoredAssertionRefV1 = t.type({
+  assertionRef: AssertionRef,
+  version: t.literal(1),
+});
+
+// StoredAssertionRefV2
+export type StoredAssertionRefV2T = t.TypeOf<typeof StoredAssertionRefV2>;
+export const StoredAssertionRefV2 = t.type({
+  assertionRef: AssertionRef,
+  version: t.literal(2),
+});
```

Comment:
This object will be stored one time for user (milions of records) on Redis. Maybe we should consider a more compact format to consume less memory. For example:
```typescript
export const StoredAssertionRefV1 = t.type({
  a: AssertionRef,
  v: t.literal(1),
});
```
Or alternatively  we can improve the encoder decoder to produce a JSON with that form (I prefer this option)